web/assets/js/widgets/core/widgets.js

```diff
@@ -387,7 +387,7 @@ window.CultuurnetWidgets = window.CultuurnetWidgets || { behaviors: {} };
           newParams.push(key + '=' + value);
         });
 
-        return newParams.join('&').replace('[', '%5B').replace(']', '%5D');
+        return newParams.join('&').replaceAll('[', '%5B').replaceAll(']', '%5D');
```

Comment:
Not sure if it's better.
But you could maybe use `encodeURIComponent('[')` ?

Reply:
I think this will have the same issue as `encodeURI` because that will also encode spaces and that is something we wanted to avoid

![Schermafbeelding 2021-04-22 om 22 01 38](https://user-images.githubusercontent.com/1322949/115844245-f10c7900-a41f-11eb-8fa0-91ad283015a5.png)

I think the method `encodeURIComponent`
> This function encodes special characters. In addition, it encodes the following characters: , / ? : @ & = + $ #

Reply:
Replace the hardcoded instances.